Testing/UnitTests/xASL_ut_master_xASL_ExploreASL_Master.m

```diff
@@ -264,7 +264,7 @@
 testTime = tic;
 
 % Copy test patient
-testPatientSource = fullfile(TestRepository,'UnitTesting','synthetic_dcm','test_patient_2_2_0'); 
+testPatientSource = fullfile(TestRepository,'UnitTesting','synthetic_dcm','test_patient_2_2_0');
```

Comment:
2_2_0 is on purpose here? Shouldn't we rename the input files?

Reply:
I once created some synthetic dicoms of the asldro 2.2.0. I don't have that from 2.3.0 yet.

Reply:
I see. Good. Can you then create a new issue for doing that in the future?

Reply:
Yep